config-connector/solutions/iam/kpt/member-iam/Kptfile

```diff
@@ -0,0 +1,63 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: .
+packageMetadata:
+  shortDescription: sample description
+openAPI:
+  definitions:
+    io.k8s.cli.setters.project-id:
+      x-k8s-cli:
+        setter:
+          name: project-id
+          value: ${PROJECT_ID?}
+          setBy: PLACEHOLDER
+      description: The target project where you want to create service accounts and
+        grant permissions.
+    io.k8s.cli.substitutions.project-id-in-sa-sub:
+      x-k8s-cli:
+        substitution:
+          name: project-id-in-sa-sub
+          pattern: serviceAccount:SERVICE_ACCOUNT_NAME_SETTER@PROJECT_ID_SETTER.iam.gserviceaccount.com
+          values:
+          - marker: PROJECT_ID_SETTER
+            ref: '#/definitions/io.k8s.cli.setters.project-id'
+          - marker: SERVICE_ACCOUNT_NAME_SETTER
+            ref: '#/definitions/io.k8s.cli.setters.service-account-name'
+    io.k8s.cli.substitutions.project-id-sub:
+      x-k8s-cli:
+        substitution:
+          name: project-id-sub
+          pattern: projects/PROJECT_ID_SETTER
+          values:
+          - marker: PROJECT_ID_SETTER
+            ref: '#/definitions/io.k8s.cli.setters.project-id'
+    io.k8s.cli.setters.policy-member-name:
```

Comment:
Is there any value in making this a setter? It seems that in the optional workflow the user will need to edit yaml files manually anyway in order to use this since resource names need to be unique within the project.

Reply:
This field is made to be a setter because we want customers to use it to change the resource name. They don't need to manually update in the yaml file. I think I explicitly described it in the optional workflow. Did you find it confusing?

Reply:
To me, a setter signals that this is something we expect the customer to want to change. I was unclear as to why the user would be changing it. It appears that this was part of the optional workflow. I'm not sure if kpt setters are going to be useful at all in the case where a customer wants to apply an arbitrary set of roles to an arbitrary set of members.